my-app/src/index.css

```diff
@@ -1,13 +1,59 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=DM+Mono:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "DM Mono", monospace, Arial, Helvetica, sans-serif;
+  font-size: 16px;
```

Comment:
@diwanow Which one you recommend, to use font-size: 62,5% or to use font-size: 10px (just to be sure that you would get 10px at the end)?

Reply:
62.5 because of accessibility - 10px is an absolute measure and might override the default browser settings for a user making text and elements potentially too small for some people; 62.5% would respect the user's settings.